yt/odin/bin/yt_odin_webservice/__main__.py

```diff
@@ -135,6 +137,20 @@ def create_solomon_timed_out_sensors(records):
     return sensors
 
 
+def get_prometheus_metric_line(
+        metric_name: str, metric_value: float, timestamp: Optional[int], labels: Optional[Dict[str, str]] = None
```

Comment:
`int | None`? `dict[str,str] | None` ?

Reply:
Would be nice. What is the minimal python version that should be supported by odin?

Reply:
I think they are not specified. Internally we use 3.12 and don't have tests for other versions.

I think we can use these type hints.